frontend/src/toolbar/elements/elementsLogic.ts

```diff
@@ -383,6 +388,21 @@ export const elementsLogic = kea<
             } else {
                 actions.setSelectedElement(element)
             }
+
+            posthog.capture('viewed toolbar element', {
+                element_tag: element?.tagName.toLowerCase(),
+                element_type: element?.type,
+                has_href: !!element?.href,
+                has_class: !!element?.className,
+                has_id: !!element?.id,
+                has_name: !!element?.name,
+                has_data_attr: !!element?.attributes.getNamedItem('data-attr'),
```

Comment:
data-attr is a posthog-specific thing, not sure what value there is in capturing it. Long-term we would like to handle all data-attributes - https://github.com/PostHog/posthog/issues/2140 (haven't got to this yet due to focus).

There's nothing special about `data-attr` - for example at my last job we used `data-js-mobile-task-link="`123"` for similar purposes.

Reply:
I guess there is value in knowing if users are using it. We do treat `data-attr` differently (it's like an `id`) and there's even [a pending issue](https://github.com/PostHog/posthog.com/issues/499) about documenting this. I have nothing against reevaluating this, though there might be some value in capturing it. Not sure... :)

Reply:
You've raised actually a good point @macobo, aside from keeping the `data-attr` handling separate as we do have special handling, I've added an attribute containing other `data-*` attrs that the element may have set. This can better help us learn if it's worth expanding support for more attributes.